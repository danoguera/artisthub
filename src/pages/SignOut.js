import React from 'react';


class SignOut extends React.Component{
    constructor(){
        super();
        this.state = {
            email: "",
            password: "",
            result: "",
        } 
    } 

    componentDidMount(){
        localStorage.removeItem("token");
        this.props.onUpdate("");
    } 

    render(){
        if (this.state.loading){return <h1>Loading...</h1>}
      

        return (
            <main>
            <div className="login"><h1>Gracias por tu visita.</h1>
            </div>
            </main>
        )

    }
} 

export default SignOut;
