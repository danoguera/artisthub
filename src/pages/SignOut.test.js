import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import SignOut from './SignOut';

describe('SignOut', () => {
  it('should render correctly', () => {
    const onUpdate = jest.fn();
    const { container } = render(<SignOut onUpdate={onUpdate}/>);

    expect(container.firstChild).toMatchSnapshot();
  });

})