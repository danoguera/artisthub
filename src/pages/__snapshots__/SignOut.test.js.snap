// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignOut should render correctly 1`] = `
<main>
  <div
    class="login"
  >
    <h1>
      Gracias por tu visita.
    </h1>
  </div>
</main>
`;
